### ISSUE TITLE
Chrome Reload / Back causes problems

### ISSUE THREAD

**Issue by michaelkrieger** [jellyfin/jellyfin-web#2526]
**Describe The Bug**
JellyFin Docker redirected with Traefik v2.
Works practically all the time.  About once every few days, I'll be on https://host:port/web/index.html#!/details?id=H0H0H0&serverId=H0H0H0 and hit refresh in the browser or use the browser's back button.  I end up on https://host:port/web#!/details?id=H0H0H0&serverId=H0H0H0 instead. That causes a 404 error.  Moreover, clicking back will continue to give me the 404 error on other pages.  Ultimately I have to clear the URL manually and everything goes to normal.
I feel like there is a cache clean going on at the same time.

**Steps To Reproduce**

Browse around the WebUI in Chrome
be on https://host:port/web/index.html#!/details?id=H0H0H0&serverId=H0H0H0
Refresh or use browser back buttons on occasion
Wait for 404 error and note that the browser is on https://host:port/web#!/details?id=H0H0H0&serverId=H0H0H0 

**Expected Behavior**
I'm not sure why the full URI is being truncated.  This has happened for years but very sporadically.

**Logs**

**Screenshots**

**System (please complete the following information):**
 - Platform: Windows (UI), Docker Image (Server)
 - Browser: Chrome 89.0.4389.90
 - Jellyfin Version: 10.7.1
 - Traefik v2
```
       - "traefik.http.routers.jellyfin.entryPoints=secure"
      - "traefik.http.routers.jellyfin.rule=Host(`jellyfin.mydomain`)" #PathPrefix(`/jellyfin`)"
      - "traefik.http.routers.jellyfin.tls=true"
      - "traefik.http.routers.jellyfin.tls.certResolver=leresolver"
      - "traefik.http.routers.jellyfin.tls.domains=jellyfin.mydomain"
      - "traefik.http.routers.jellyfin.middlewares=jellyfin-mw"
      - "traefik.http.middlewares.jellyfin-mw.headers.customResponseHeaders.X-Robots-Tag=noindex,nofollow,nosnippet,noarchive,notranslate,noimageindex"
      - "traefik.http.middlewares.jellyfin-mw.headers.SSLRedirect=true"
      - "traefik.http.middlewares.jellyfin-mw.headers.SSLHost=jellyfin.mydomain:9999"
      - "traefik.http.middlewares.jellyfin-mw.headers.SSLForceHost=true"
      - "traefik.http.middlewares.jellyfin-mw.headers.STSSeconds=315360000"
      - "traefik.http.middlewares.jellyfin-mw.headers.STSIncludeSubdomains=true"
      - "traefik.http.middlewares.jellyfin-mw.headers.STSPreload=true"
      - "traefik.http.middlewares.jellyfin-mw.headers.forceSTSHeader=true"
      - "traefik.http.middlewares.jellyfin-mw.headers.frameDeny=true"
      - "traefik.http.middlewares.jellyfin-mw.headers.contentTypeNosniff=true"
      - "traefik.http.middlewares.jellyfin-mw.headers.browserXSSFilter=true"
      - "traefik.http.middlewares.jellyfin-mw.headers.customFrameOptionsValue='allow-from https://mydomain'"
      - "traefik.http.routers.jellyfin.service=jellyfin-svc@file"
      - "traefik.http.services.jellyfin-svc.loadBalancer.server.port=8096"
      - "traefik.http.services.jellyfin-svc.loadBalancer.passHostHeader=true"
      - "traefik.http.routers.jellyfin-insecure.entryPoints=secure"
      - "traefik.http.routers.jellyfin-insecure.rule=Host(`jellyfin.mydomain`)"
      - "traefik.http.routers.jellyfin-insecure.middlewares=jellyfin-insecure-mw"
      - "traefik.http.middlewares.jellyfin-insecure-mw.redirectscheme.scheme=https"
      - "traefik.http.middlewares.jellyfin-insecure-mw.redirectscheme.port=9999"
      - "traefik.http.middlewares.jellyfin-insecure-mw.redirectscheme.permanent=false"
      - "traefik.http.routers.jellyfin-insecure.service=noop@internal"
```

**Additional Context**
Oddly, looking at the Chrome devtools, it shows many many request to https://host:port/web/auto[CODE].woff2 which come back as a 404 triggered by the serviceworker.js.  Not sure if it's related.

**Comment by michaelkrieger** [comment]
Still occurs from time to time.

**Comment by sambartik** [comment]
Can confirm that this also happens in Firefox, so it is not something browser related.

**Comment by michaelkrieger** [comment]
So I've managed to do this reproducibly on chrome with a clean 10.7.7 installation.  Even captured in on video doing it multiple times.  Happy to send the screen recording over to someone (though it does have URLs and whatnot in it, so would prefer to not publish it).

Basically, if I go into my Movies library and click on a movie, and then immediately (before it loads) press F5, I'll get the 404 error.  Back on the browser then doesn't solve things until I get to the root folder.

When I do this, it goes from my normal browsing ( https://host:port/web/index.html#!/list.html?parentId=...&serverId=... ) to  https://host:port/web in the address bar.  Meanwhile the text in the browser/chome shows ( No webpage was found for the web address:  https://host:port/web#!/list.html?parentId=...&serverId=... ) which you'll notice is a very different URL.
Pushing back again takes me to https://host:port/web.  Doing it again takes me to the then-working https://host:port/web/index.html#!/home.html

**So the question is, why is it updating a baseURL from /web/index.html#!/list.html to /web#!/list.html ?** And so if you refresh or do something at just the right time, it ends up loading the wrong URL?

**Comment by michaelkrieger** [comment]
Bumpity

**Comment by michaelkrieger** [comment]
This issue remains:
This jellyfin.mydomain.com page can’t be found
No webpage was found for the web address: https://jellyfin.mydomain.com:9876/web#!/details?id=8...c&serverId=9...4
HTTP ERROR 404

To do this, I edited a show, clicked the three dots, clicked refresh, and then pushed F5.
Clicking back in the browser takes me through a variety of pages:
 https://jellyfin.mydomain.com:9876/web#!/dashboard.html
until I get to
https://jellyfin.mydomain.com:9876/web/index.html#!/home.html

Basically, it is losing the "/index.html" and thus returning a 404 not found.  This seems to happen after a refresh

**Comment by michaelkrieger** [comment]
Lately I've been seeing the issue every time I refresh metadata in the Jellyfin iOS app as well. Go into a movie, refresh metadata. Choose "replace all" and go. You can no longer navigate away from that page as the back button doesn't work. Meanwhile if you can get it to go anywhere you get a 404 error with the curious path again. Clearly Jellyfin is mucking up a path on updates both on the iOS app and Jellyfin Web.

This seems common and reproducible.

![image](https://user-images.githubusercontent.com/7321259/177918786-3bdc5935-431c-45f2-a472-382df1b3177e.jpeg)

**Comment by amanaedk** [comment]
I'm getting this a lot on 10.8.1 with Firefox, in particular every time I edit metadata for a music track.

**Comment by michaelkrieger** [comment]
Confirming still an issue in 10.8.10 (and also a bump).  This impacts the ios app that wraps jellyfin-web.

**Comment by herigrosir** [comment]
1.go to dashboard and go to scheduled taks and clean your cache
2.youtur ip server mush be to lock or change to ip static not dynamic
3.client in lokal area netword to much , there may be too many clients on your local network
until the network is busy
4 uninstall and install again jellyfin server 

if you have a lot of clients it's better
Just create a server with a shared or SMB folder
clinet just use VLC, connect to local VLC , no need jellyfin and medatada
but file movie musbe .mkv

Maybe that's the best solution, just try it

**Comment by thornbill** [comment]
This is likely fixed in 10.9 since `/web` will now load the index file by default.
